src/app/components/utility/modal/modal.component.ts

```diff
@@ -207,6 +244,36 @@ export class ModalComponent implements OnInit {
       } else {
         this.isDisabled = true;
       }
```

Comment:
Simplify in all the places like this:
`  this.isDisabled = e.target.value !== this.challenge.title;
`

Reply:
done